### PR TITLE
translate datastore

### PR DESCRIPTION
**Pull request by cw123**
### What problem does this PR solve?

Problem Summary: Translate datastore code comments from Chinese to English

### What is changed and how it works?

What's Changed: Translate datastore code comments

How it Works: Translate datastore code comments

Side effects(Breaking backward compatibility? Performance regression?):  No

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: src/chunkserver/datastore/chunkserver_chunkfile.cpp [hunk 1]
```diff
@@ -736,23 +774,32 @@ bool CSChunkFile::needCreateSnapshot(SequenceNum sn) {
...
-    // 又因为snapSn正常情况下是小于metaPage_.sn，所以snapSn也应当小于sn
-    // 有几种情况可能出现metaPage_.sn <= snap.sn
-    // 场景一：DataStore重启恢复历史日志，可能出现metaPage_.sn==snap.sn
-    // 重启前有一个请求产生了快照文件，但是还没更新metapage时就重启了
-    // 重启后又回放了先前的一个操作，这个操作的sn等于当前chunk的sn
-    // 场景二：follower在做raft的恢复时通过leader下载raft快照
-    // 下载过程中，Leader上的chunk也在做chunk的快照，下载以后follower再做日志恢复
-    // 由于follower先下载chunk文件，后下载快照文件，所以此时metaPage_.sn<=snap.sn
+    // The preceding logic ensures that the sn here must be equal to metaPage.sn
+    // Because sn<metaPage_.sn, the request will be rejected
```
Comment (bai-charisu) [inline src/chunkserver/datastore/chunkserver_chunkfile.cpp:785]: Because后面加个if

**Review by bai-charisu — APPROVED** [review]

File: src/chunkserver/datastore/chunkserver_chunkfile.cpp [hunk 2]
```diff
@@ -110,13 +110,13 @@ CSErrorCode ChunkFileMetaPage::decode(const char* buf) {
...
-    // 校验crc，校验失败返回错误码
+    // check crc
     if (crc != recordCrc) {
         LOG(ERROR) << "Checking Crc32 failed.";
-        return CSErrorCode::CrcCheckError;  // 需定义crc校验失败的错误码
+        return CSErrorCode::CrcCheckError;
     }
 
-    // TODO(yyk) 判断版本兼容性，当前简单处理，后续详细实现
+    // TODO(yyk) check version compatibility, need detailed implementation later
```
[the remark this answers is not in the document]
Comment (cw123) [reply src/chunkserver/datastore/chunkserver_chunkfile.cpp:119]: // TODO(yyk) check version compatibility, currrent simple error handing,
    // need detailed implementation later

File: src/chunkserver/datastore/chunkserver_chunkfile.cpp [hunk 3]
```diff
@@ -221,7 +224,8 @@ CSErrorCode CSChunkFile::Open(bool createFile) {
     }
 
     CSErrorCode errCode = loadMetaPage();
-    // 重启后，只有重新open加载metapage后，才能知道是否为clone chunk
+    // After restarting, only after reopening and loading the metapage,
+    // can we know whether it is a clone chunk
```
[the remark this answers is not in the document]
Comment (cw123) [reply src/chunkserver/datastore/chunkserver_chunkfile.cpp:228]: only+状语提前，句子部分倒装

File: src/chunkserver/datastore/chunkserver_chunkfile.cpp [hunk 4]
```diff
@@ -279,8 +283,10 @@ CSErrorCode CSChunkFile::Write(SequenceNum sn,
                    << ", chunk size: " << size_;
         return CSErrorCode::InvalidArgError;
     }
-    // 用户快照以后会保证之前的请求全部到达或者超时以后才会下发新的请求
-    // 因此此处只可能是日志恢复的请求，且一定已经执行，此处可返回错误码
+    // After snapshot, will ensure that all previous requests arrive or time out
+    // before issuing new requests. Therefore, this is only a log recovery
+    // request, and it must have been executed, and an error code can be
+    // returned here.
```
[the remark this answers is not in the document]
Comment (cw123) [reply src/chunkserver/datastore/chunkserver_chunkfile.cpp:289]: done

File: src/chunkserver/datastore/chunkserver_chunkfile.cpp [hunk 5]
```diff
@@ -560,8 +570,9 @@ CSErrorCode CSChunkFile::Delete(SequenceNum sn)  {
         return CSErrorCode::BackwardRequestError;
     }
 
-    // 如果存在快照，就先删除快照，正常是不会出现这种调用的
-    // Delete语义就是将chunk删除，不管存不存在快照
+    // If there is a snapshot, delete the snapshot first,
+    // normally there will be no such situation
+    // delete the chunk, regardless of whether the snapshot exists or not
```
[the remark this answers is not in the document]
Comment (cw123) [reply src/chunkserver/datastore/chunkserver_chunkfile.cpp:575]: done

File: src/chunkserver/datastore/chunkserver_chunkfile.cpp [hunk 6]
```diff
@@ -596,15 +607,17 @@ CSErrorCode CSChunkFile::Delete(SequenceNum sn)  {
 CSErrorCode CSChunkFile::DeleteSnapshotOrCorrectSn(SequenceNum correctedSn)  {
     WriteLockGuard writeGuard(rwLock_);
 
-    // 如果是clone chunk， 理论上不应该会调这个接口，返回错误
+    // If it is a clone chunk, theoretically this interface should not be called
+    // and an error is returned
```
[the remark this answers is not in the document]
Comment (cw123) [reply src/chunkserver/datastore/chunkserver_chunkfile.cpp:611]: done

**Review by ilixiaocui — APPROVED** [review]

**Pull request merged**